providers/base/units/miscellanea/test-plan.pxu

```diff
@@ -34,6 +34,7 @@ include:
     miscellanea/debsums                            certification-status=blocker
     miscellanea/check_prerelease                   certification-status=blocker
     miscellanea/ubuntu-desktop-recommends          certification-status=blocker
+    miscellanea/grub_file_check
```

Comment:
How critical is this test? Can we issue certificates if this file is missing (that is, if the test fails)?

Reply:
@linjuiche Hi Kent, what do you think? should this be a certification blocker case?